ios/FluentUI.Demo/FluentUI.Demo/Demos/ThemingDemoController.swift

```diff
@@ -68,21 +68,18 @@ class ThemingDemoController: DemoController {
         let overridingTheme = CustomStyle()
         let customThemeButtonPrimary = MSFButton(style: .primary,
                                                       size: .medium,
-                                                      theme: overridingTheme,
```

Comment:
Can you remind me how you're overriding the tokens directly in the control level?
Can you also provide videos or screenshots in the validation so we know the visual results of those changes?

Reply:
Control-level overrides are done by setting the `overrideTokens` property directly on the control in question. See here for an example in `ThemingDemoController`:

https://github.com/microsoft/fluentui-apple/blob/b872a8025116ac22f80d4a5dc77fb79603752714/ios/FluentUI.Demo/FluentUI.Demo/Demos/ThemingDemoController.swift#L108